.circleci/config.yml

```
@@ -397,6 +397,9 @@ workflows:
               - master
               - develop
               - /dependabot/.*/bevrand.uitests/.*/
+              - feature/*uitests*
```

Syntax is incorrect, any character n times is written with . (any character) *(any amount of time) 
```suggestion
              - /feature/.*uitests.*/
```
Should also start with a / for some reason I believe